src/libraries/System.Utf8String.Experimental/src/System/Runtime/Intrinsics/Intrinsics.Shims.cs

```diff
@@ -26,6 +26,18 @@ internal static class Vector128
 
 namespace System.Runtime.Intrinsics.X86
 {
+    internal abstract class X86Base
+    {
+        public abstract class X64
+        {
+            public const bool IsSupported = false;
+            public static ulong BitScanForward(ulong value) => throw new PlatformNotSupportedException();
```

Comment:
This should probably be `internal` to match the `X86Base`...

Reply:
Yeah, the X86Base in this shim will always be internal so it doesn't really matter, but it wouldn't hurt to match the actual API better. Will tidy this up.